docs/endpoints/summary-endpoints.md

```diff
@@ -22,12 +22,13 @@ The following endpoints are for retrieving and managing UID2 tokens (identity to
 | [POST&nbsp;/token/validate](post-token-validate.md) | Used for testing, to validate that an advertising token (UID2) matches the specified email address, phone number, or the respective hash. | Required | Required |
 | [POST&nbsp;/token/refresh](post-token-refresh.md) | Generate a new token for a user for their refresh token from the [POST&nbsp;/token/generate](post-token-generate.md) response. | N/A | Required |
 
-## Identity Maps
+## Identity Map and Opt-Out
 
 The following endpoints are used by advertisers and third-party data providers. Publishers do not need to use these endpoints.
```

Comment:
If we add opt-out to this section, we might need to update the text as /optout/status is also for DSPs and sharers.

Alternatively we could put opt-out in its own section. I'm good either way.

Reply:
@mcollins-ttd very good point, thx. Making it into a new section as you originally had it.